libraries/SITL/SIM_JSON.h

```diff
@@ -124,10 +121,25 @@ class JSON : public Aircraft {
         { "", "rng_4", &state.rng[3], DATA_FLOAT, false },
         { "", "rng_5", &state.rng[4], DATA_FLOAT, false },
         { "", "rng_6", &state.rng[5], DATA_FLOAT, false },
+        {"","velocity_wind", &state.velocity_wind, DATA_VECTOR3F, false},
         {"windvane","direction", &state.wind_vane_apparent.direction, DATA_FLOAT, false},
         {"windvane","speed", &state.wind_vane_apparent.speed, DATA_FLOAT, false},
         {"", "airspeed", &state.airspeed, DATA_FLOAT, false},
         {"", "no_time_sync", &state.no_time_sync, BOOLEAN, false},
+        { "rc", "rc_1", &state.rc[0], DATA_FLOAT, false },
+        { "rc", "rc_2", &state.rc[1], DATA_FLOAT, false },
+        { "rc", "rc_3", &state.rc[2], DATA_FLOAT, false },
+        { "rc", "rc_4", &state.rc[3], DATA_FLOAT, false },
+        { "rc", "rc_5", &state.rc[4], DATA_FLOAT, false },
+        { "rc", "rc_6", &state.rc[5], DATA_FLOAT, false },
+        { "rc", "rc_7", &state.rc[6], DATA_FLOAT, false },
+        { "rc", "rc_8", &state.rc[7], DATA_FLOAT, false },
+        { "rc", "rc_9", &state.rc[8], DATA_FLOAT, false },
+        { "rc", "rc_10", &state.rc[9], DATA_FLOAT, false },
+        { "rc", "rc_11", &state.rc[10], DATA_FLOAT, false },
+        { "rc", "rc_12", &state.rc[11], DATA_FLOAT, false },
+        { "battery", "voltage", &state.bat_volt, DATA_FLOAT, false },
+        { "battery", "current", &state.bat_amp, DATA_FLOAT, false },
```

Comment:
Should have a index, we may want to support more in the future.

Reply:
unfortunately that wouldn't work with SimNet in a backwards compatible way (quite a few users have existing ArduPilot firmware using SimNet, this PR is aiming to allow them to use master/stable etc)
Given we've had the JSON backend for 5 years without any battery support at all, and no other SITL backend supports more than one battery (the SIM_Aircraft class hard codes one battery), I think it is reasonable to go with a single battery to support backwards compatibility